routes/shopItemByStatus.js

```diff
@@ -1,8 +1,24 @@
 const express = require("express");
+const utility = require("../utility.js");
 
 findStatusRouter = express.Router();
 
-findStatusRouter.get("/", (req,res,next) => {
+findStatusRouter.use(utility.readDatabaseMiddleware);
+
+findStatusRouter.get("/", (req, res, next) => {
+    const isQuery = req.query.isCompleted || "";
```

Comment:
`isCompleted` would be a better name for this. Maybe there will be multiple query parameters later.